os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -43,19 +43,15 @@ static void ns_flash_erase(uint32_t address)
 {
 	flash_t flash;
 
```

Comment:
Please check if `#include "device_lock.h"` is required. Else pls remove it.

Reply:
It is needed, device_mutex_unlock() is used at the bottom of file.